auth/rbac.go

```diff
@@ -137,30 +138,31 @@ func (rc *rbacClientImpl) requestAccess(url, identityToken string) []types.RbacD
 
 // aggregatePermissions loop over all the permissions/roles/alcs of the user returned
 // from RBAC and creates and return the map of permissions where key is
-// resourceType (openshift.cluster, openshift.node, openshift.project) and the values are the
+// resourceType (recommendation-results) and the values are the
 // slice of resources (cluster names, node names, project names).
+// We are interested in this ACLs: https://github.com/RedHatInsights/rbac-config/blob/master/configs/prod/permissions/ocp-advisor.json
 func aggregatePermissions(acls []types.RbacData) map[string][]string {
 	permissions := map[string][]string{}
 	for _, acl := range acls {
-		resourceType := strings.Split(acl.Permission, ":")[1]
-		if strings.Contains(resourceType, "openshift") {
+		splits := strings.Split(acl.Permission, ":")
+		if len(splits) < 3 {
+			log.Warn().Str("ACL", acl.Permission).Msg("Unexpected RBAC response")
+			continue
+		}
+		if splits[0] != "ocp-advisor" {
+			// check the ACL is for ocp-advisor, not for other APIs
+			continue
+		}
+		resourceType := splits[1]
+		verb := splits[2]
```

Comment:
what's this var representing?

Reply:
the ACLs has this format: `API:RESOURCE:VERB`, for example `ocp-advisor:read-recommendations:read`, so the `verb` variable is the action the SA can make on the given resource for the given API: https://console.redhat.com/docs/api/rbac/v1#operations-Access-getPrincipalAccess